packages/sui-hoc/src/withOpenToggle.js

```diff
@@ -5,7 +5,7 @@ export default BaseComponent => {
 
   return class WithOpenToggle extends Component {
     state = {
-      isOpen: false
+      isOpen: Boolean(this.props.isOpen) // eslint-disable-line
```

Comment:
Why the eslint comment? 😮 Anyway, could `!!` be used instead?

```suggestion
      isOpen: !!this.props.isOpen
```

Reply:
![Screen Shot 2020-11-27 at 13 28 24](https://user-images.githubusercontent.com/1307927/100449593-7cbdeb80-30b4-11eb-8dbe-1646596af69c.png)

Reply:
You can do something more specific: `// eslint-disable-line react/prop-types`

Reply:
The bundle is not affected by adding PropTypes as we're removing them for Pro bundles.

Reply:
@midudev @desko27 @kikoruiz PropTypes added and I removed the eslint comment